vnext/Microsoft.ReactNative/Views/Image/ImageViewManager.h

```diff
@@ -18,7 +18,11 @@ class ImageViewManager : public FrameworkElementViewManager {
       const winrt::Microsoft::ReactNative::IJSValueWriter &writer) const override;
   void GetNativeProps(const winrt::Microsoft::ReactNative::IJSValueWriter &writer) const override;
   ShadowNode *createShadow() const override;
-  void EmitImageEvent(xaml::Controls::Grid grid, const char *eventName, ReactImageSource &source);
+  void EmitImageEvent(
+      xaml::Controls::Grid grid,
+      const char *eventName,
+      ReactImageSource &source,
+      const winrt::hstring &errorMessage = L"");
```

Comment:
>winrt::hstring [](http://example.com/codeflow?start=12&length=14)

Knowing nothing about the semantics, just curious: Would an HRESULT give us what we need, with potentially lower initialization and processing costs?

Reply:
The error codes come from LoadedImageSourceLoadStatus, SvgImageSourceLoadStatus enums and ExceptionRoutedEventArgs.ErrorMessage(), it doesn't map to HRESULT well